buy-resell/products/3rd-party/datev-meinfiskal.md

```diff
@@ -52,25 +52,28 @@ As a PosDealer, you can get an overview of all your PosSystems and their **PosSy
 1. Log in to **fiskaltrust.Portal** and select `PosSystems`.
 2. If no PosSystem should be available, contact your PosCreator.
 
-The PosOperator already has an account in the **fiskaltrust.Portal**, agreed to the general terms and conditions and the PosOperator user agreement of fiskaltrust, checked his master data, and you, as a PosDealer, have the authorization to sign contracts. If these requirements are not met, the [PosOperator Onboarding](../../../getting-started/operator-onboarding/invitation-process.md) must be completed first, or the PosOperator itself must perform the setup.
+If the following requirements are not met, the [PosOperator Onboarding](../../../getting-started/operator-onboarding/invitation-process.md) must be completed first, or the PosOperator itself must perform the setup.
+
+1. The PosOperator already has an account in the **fiskaltrust.Portal** and agreed to the general terms and conditions and the PosOperator user agreement of fiskaltrust.  
+2. The [Master data](../../../getting-started/operator-onboarding/master-data.md "Master data") are checked by the PosOperator or by the PosDealer.  
+3. As a PosDealer, you have full authorizations (Write/Read, Contract Conclusion) to the account of the PosOperator.  
 
 ### Sign contract permission
 
   1. Log in to **fiskaltrust.Portal** as a PosDealer. 
   2. Go to `PosOperator` / `Overview`. 
   3. If necessary, enter filter criteria to narrow the search results and select `Search`. 
-  4. Check with the icon at **Permissions** if `sign contract` is active.
-  5. If this permission is not active, contact the PosOperator to give it to you.
+  4. Check with the icon at **Permissions** if `Contract conclusion` is active.
+  5. If this permission is not active, contact the PosOperator to activate it to you.
   6. Close the dialog box by clicking **OK**. 
   
 ###  Master data
 
   1. Select the link at `Name` and go to the account of the PosOperator.
   2. Select `Company` / `Master data`.
   3. Check if every mandatory field, like `Name*` or `Address*`, is filled in.
-
-  5. Check whether you can successfully perform a validity check with either St.-ldNr. or USt-ldNr.
-  6. Save your entries with **Save**. 
+  4. Check whether you can successfully perform a validity check with either `St.-ldNr.` or `USt-ldNr`.

```

Comment:
It's St-ldNr in the portal without the "." - would change it either here or there and if there also the USt-ldNr